overlord/assertstate/validation_set_tracking.go

```diff
@@ -104,8 +104,26 @@ func UpdateValidationSet(st *state.State, tr *ValidationSetTracking) {
 	st.Set("validation-sets", vsmap)
 }
 
+// isForgetAllowedByModelAssertion checks whether a validation-set is controlled by
+// the model assertion. If the validation-set is set to 'enforce', then it's not possible
+// to forget it.
+func isForgetAllowedByModelAssertion(st *state.State, accountID, name string) error {
+	vs, err := validationSetFromModel(st, accountID, name)
+	if err != nil {
+		return err
+	}
+	if vs == nil {
+		return nil
+	}
+	if vs.Mode != asserts.ModelValidationSetModePreferEnforced {
```

Comment:
shouldn't this be == ModelValidationSetModeEnforced instead?

Reply:
Good point